src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css

```diff
@@ -0,0 +1,95 @@
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 50px 0;
```

Comment:
Why does this have this **HUGE** padding ? This is too much

![Screenshot from 2021-03-12 12-21-37](https://user-images.githubusercontent.com/1193222/110926841-8ef32b00-832d-11eb-8d1e-30bcd2ac86e2.png)

Reply:
Yeah, sorry about that. This is something I forgot to remove, from when I was testing things